modules/get-started/pages/licensing/overview.adoc

```diff
@@ -36,14 +36,47 @@ The Community Edition is licensed with the Redpanda https://github.com/redpanda-
 == Enterprise Edition
 
 The Enterprise Edition is licensed with the https://github.com/redpanda-data/redpanda/blob/dev/licenses/rcl.md[Redpanda Community License^] (RCL). It includes the free features licensed under the Redpanda BSL, as well as enterprise features.
+You can also generate a <<Trial license>> for the Enterprise Edition.
+
+=== Trial license
+
+All new Redpanda clusters automatically receive a trial license that's valid for 30 days. If you wish to extend this trial an additional 30 days, use the xref:reference:rpk/rpk-generate/rpk-generate-license.adoc[`rpk generate license`] command. This extended evaluation period begins after you apply the trial license to your cluster using the `--apply` flag. After this period expires, inactive enterprise features are disabled, and active features enter a restricted state.
+
+IMPORTANT: Only one trial license is permitted per email and business domain. Attempts to generate multiple trial licenses from the same email or business domain result in a license error. To get a permanent license, contact https://www.redpanda.com/contact[Redpanda Sales^].
+
+To generate and apply a 30-day trial license for Enterprise Edition:
+
+[source,bash,subs="attributes+"]
+----
+
+rpk generate license --apply
+First Name: <your_first_name>
+Last Name: <your_last_name>
+Company name: <your company name>
+Email: <your business email address>
+
+Successfully saved license to "/home/<your local directory>/code/rp/redpanda/src/go/rpk/redpanda.license".
+
+Upload this license in Redpanda Console, or run:
+  rpk cluster license set --path /home/<your local directory>/rp/redpanda/src/go/rpk/redpanda.license
+
+This license expires on 2025-05-07.
+
+For more information, see:
+  https://docs.redpanda.com/current/get-started/licensing/overview/#license-keys
```

Comment:
Please could we move this out into a separate ' Example output' block? This will allow users to copy/paste the command.